src/components/DefineCompany/YourCompany.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+  <v-card flat>
+     <div class="define-company-header">
+        <v-icon>mdi-domain</v-icon>
+        <label class="define-company-title"><strong>Your Company</strong></label>
+    </div>
+
+    <div class="section-container px-8">
+      <!--TODO: Replace container content with Name Request Summary when it is ready -->
+      <v-layout row>
+        <v-flex md4>
+          <label><strong>Company Name</strong></label>
+        </v-flex>
+        <v-flex md6>
+          <div class="company-name">{{ companyName }}</div>
+          <div class="company-type">
+            <span v-if="entityFilter(EntityTypes.BCOMP)">BC Benefit Company</span>
+            <span v-else-if="entityFilter(EntityTypes.COOP)">BC Cooperative Association</span>
+          </div>
+        </v-flex>
+        <v-flex md2 class="align-right">
+          <!-- only show buttons for named company -->
+          <template v-if="getApprovedName || true">
+            <v-btn
+              v-if="!companyNameChanges"
+              text color="primary"
+              id="btn-correct-company-name"
+              @click="companyNameChanges = true"
+            >
+              <v-icon small>mdi-pencil</v-icon>
+              <span>Correct</span>
+            </v-btn>
+            <v-btn
+              v-if="companyNameChanges"
+              text color="primary"
+              id="btn-undo-company-name"
+              @click="companyNameChanges = false"
+            >
+              <v-icon small>mdi-undo</v-icon>
+              <span>Undo</span>
+            </v-btn>
+          </template>
+        </v-flex>
```

Comment:
replaced the above two `<v-flex>` with one via a conditional to house the selection menu, was what i did during development.